src/crypto/public_key/elliptic/oid_curves.js

```diff
@@ -268,8 +277,23 @@ async function validateStandardParams(algo, oid, Q, d) {
   return true;
 }
 
+/**
+ * Check whether the public point has a valid encoding.
+ * NB: this function does not check e.g. whether the point belongs to the curve.
+ */
+function checkPublicPointEnconding(oid, V) {
+  const curveName = oid.getName();
+  const { payloadSize, wireFormatLeadingByte } = curves[curveName];
```

Comment:
This is a bit nitpicky but it would be better to copy these parameters to the curve object in the`CurveWithOID` constructor, like it does for the other parameters, and then pass the `curve` object here instead of the `oid`. That way, you don't have to get the name and look up the params by name again.